README.md

```diff
@@ -3,241 +3,65 @@
 
 # The Chemistry Development Kit (CDK)
  
-Copyright 1997-2018 The CDK Development Team
+Copyright &copy; 1997-2018 The CDK Development Team
 
 License: LGPL v2, see doc/lgpl.license
 
-## Introduction
-
-You are currently reading the README file for the Chemistry Development Project (CDK).
-This project is hosted under http://cdk.github.io/
-Please refer to these pages for updated information and the latest version of the CDK. CDK's API documentation is available though our [Github site](http://cdk.github.io/cdk/).
-
-The CDK is an open-source library of algorithms for structural chemo- and bioinformatics, implemented in 
-the programming language Java(tm). The library is published under terms of the the 
-GNU Lesser General Public License v2. This has implications on what you can do with sources and
-binaries of the CDK library. For details, please refer to the file LICENSE, which should have been
-provided with this distribution.
-
-**PLEASE NOTE**: This is a library of useful data structures and algorithms to manipulate them 
-from the area of structural chemo- and bioinformatics. As such, it is intended for the use by
-programmers, who wish to save some effort by reusing code. It is not intended for the enduser. 
-If you consider yourself to be more like user, you might not find what you wanted. 
-Please refer to other projects like the JChemPaint project (http://jchempaint.github.com/)
-or the Jmol project (http://www.jmol.org/) for programs that actually take advantage of the 
-CDK library.
-
-## Compiling
```

Comment:
Looks good, but I really like to to have build instructions as part of the repo. Really, wikis get lost all the time, and people downloading an archive should not have to rely on other information to figure out how to build it. I know all those install info is trivial now, but it won't be in a few years.

Please consider moving what you removed from the README.md into an INSTALL.md file.

Reply:
Build instructions are still there.

Reply:
Under "INSTALL", could also add an INSTALL.md that goes into more details.

Reply:
This link is easiest to read: https://github.com/cdk/cdk/blob/ad91090df348c858532e70e6677e6d4d200d876b/README.md.

The issue in the past was tests used to fail so we had to have a lot of explanation about skipping tests etc. Now you just type ``mvn install`` and if it doesn't work something is wrong.

Reply:
Yes, that's true. But I was thinking indeed about the less ideal situation... where things don't build well. They do now (have you already tried Java11, btw?), but those tricks of how to compile/install/test stuff when the situation is no longer ideal (likely in a few years; or different platform), that is still relevant (as it has been for many years). This is the developer talking that has seen many perfect solutions break down over the years, and people struggling with new build systems :)

So, an INSTALL.md with some of the original tips, hints, etc would still be welcome. I'll merge in this one now.